vmdb/lib/tasks/test_metrics.rake

```diff
@@ -4,7 +4,7 @@
 #   original here: https://github.com/ManageIQ/metric_fu/blob/master/tasks/metric_fu.rake
 require_relative "./evm_test_helper"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
It's because on an appliance without test gems, then RSpec is not defined.  The *other* test tasks will not work so those *require* the check.  However, letting some test tasks through looks weird (looks like a bug) if they are seen in `rake -T`, so I decided to gate them *all* on RSpec.